### DISCUSSION TITLE
Будет ли добавлен функционал смены слоя карт на схема\спутник\гибрид\панорама

### DISCUSSION THREAD

**Discussion by Misha6787** [yandex-maps-unofficial/vue-yandex-maps#225] [Q&A (2.x)] [answered]
Добрый день, я обновлял библиотеку с beta до 2.0 в своем Vue3\Nuxt3 проекте и столкнулся с тем, что у данной версии карт, нету возможности добавить кнопку переключения слоев, как в этом примере - 
![image](https://github.com/yandex-maps-unofficial/vue-yandex-maps/assets/59843351/7334af05-84cb-4412-a806-28ccaec381f0)

Если такого функционала действительно нет, то когда он планируется добавиться и планируется ли он вообще?

**Comment by daniluk4000** [comment, marked as answer]
Добрый день. К сожалению, данный функционал не реализован самим Яндексом. У нас есть разве что [YandexMapDefaultSatelliteLayer](https://yandex-maps-unofficial.github.io/vue-yandex-maps/components/layer-default-satellite.html), который надо добавлять при старте проекта, и он даже официально у Яндекса нигде кроме справочника не упоминается в документации, и работать может криво.

Вы можете написать Яндексу в [форму обратной связи](https://yandex.ru/maps-api/support) и запросить эту фичу. Других предложений у меня для вас, увы, нет.

**Reply by Misha6787** [reply]
Понял, но разве в прошлых версиях не было этого функционала? Они его только в новых версиях не добавили?

**Reply by daniluk4000** [reply]
Ага. В новой версии они много чего не добавили из старых) но в использовании новые карты удобнее

**Reply by Misha6787** [reply]
Неприятно конечно... напишу им, узнаю, может все же добавят в будущих патчах, просто не очень хочется уходить на более старшую версию, но начальство хочет видеть функционал переключения слоев, и тут сопротивляться уже будет бесполезно

В любом случае спасибо за ответ)

**Comment by Misha6787** [comment]
> Ага. В новой версии они много чего не добавили из старых) но в использовании новые карты удобнее
